src/Avalonia.Themes.Default/Controls/TransitioningContentControl.xaml

```diff
@@ -0,0 +1,20 @@
+<Styles xmlns="https://github.com/avaloniaui"
+        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml">
+  <Style Selector="TransitioningContentControl">
+    <!-- Set Defaults -->
+    <Setter Property="Template">
+      <ControlTemplate>
+        <ContentPresenter Name="PART_ContentPresenter"
```

Comment:
Should this be named with "PART_" prefix? I don't see it used in code-behind.

Reply:
yes, its used in the ContentControl base class, it expects this part to be present in the template.

Reply:
> User might want to style template so it's good practice to name that part, especially ContentPresenter

Yes, it's good to include a name. However, the "PART_" prefix is traditionally reserved for template parts used in code-behind. If it is only named for styles, it should just be named "ContentPresenter"